2001-3000/LIT2365Storyo.xml

```diff
@@ -52,7 +44,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
-         <abstract><p>The Story of Melchizedek is an apocryphal account of the life of <persName ref="PRS6988Melchize"/>, an originally Greek text (spuriously attributed to <persName ref="PRS2209Athanasi"/>  in a
+         <abstract><p>The Story of Melchizedek is an apocryphal account of the life of <persName ref="PRS6988Melchize"/>, an originally Greek text (spuriously attributed to <persName ref="PRS2209Athanasi"/>in a
```

Comment:
I think the space needs to stay. Otherwise it will be printed "Athanasiusin".

Reply:
ok!